libseccomp/src/lib.rs

```diff
@@ -616,6 +658,15 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter or the number of syscall
     /// is invalid, an error will be returned.
```

Comment:
Could you add a empty line?

Reply:
Dear @ManaSugi san, Added the empty line.